### DISCUSSION TITLE
Pycharm logs and GitHub repository shows old login when I try to commit

### DISCUSSION THREAD

**Discussion by PenthagonHacker** [community/community#22677] [New to GitHub]
<p>Hello! I’ve deleted an old github account completely and created a new one.<br>
I am new to all this git add git commit thingy but i am doing my best.<br>
I’ve managed to create a commit via pycharm using git clone, git add , git commit and git push combination, but what i’ve noticed is that in pycharm logs and in my repository on github there’s always:</p>
<blockquote>
<p>ArtemisEntrery authored and ArtemisEntrery committed</p>
</blockquote>
<p>and this is my old account name that i have successfully deleted.</p>
<p>Can you please help me with this problem?<br>
Thanks!</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Git records an author name and email address whenever you make a commit. That is configured locally, so maybe the data there is still from when you were using your old account? Here’s the Git Book <a href="https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup#_your_identity">section on how to configure your identity</a>, that should do what you need in principle.</p>
<p>Two Github-specific notes, though:</p>
<ol>
<li>
<p>Github uses the author email to detect that you made the commit and connect it to your account. If you want that (I assume you do), you need to either use an email address that’s on your account, or if you want to keep your address private you can use a Github-provided no-reply address, see <a href="https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/setting-your-commit-email-address#about-commit-email-addresses">About commit email addresses</a>. If Github recognizes the address it always shows your username instead of whatever author name is in the commit.</p>
</li>
<li>
<p>It is technically possible to <a href="https://docs.github.com/en/github/using-git/changing-author-info">change the author information in old commits</a>, but that will cause chaos if anyone else is using the repository, because it changes history.</p>
</li>
</ol>
